SQL/Archive/17.1/2017-03-15-Disclaimer.sql

```diff
@@ -0,0 +1,2 @@
+CREATE TABLE `disclaimer` (`module` varchar(100) NOT NULL, `value` TEXT DEFAULT NULL);
```

Comment:
If we use pselectOne are we assuming there can be only 1 disclaimer per module? If yes, a unique/primary key would be awesome